src/SearchResults.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import moment from "moment"
+
+
+
+const SearchResults = props => {
+    return (
+        <table className="table table-hover table-dark">
+            <thead>
+                <tr>
+                    <th scope="col">id</th>
+                    <th scope="col">title</th>
+                    <th scope="col">firstName</th>
+                    <th scope="col">surname</th>
+                    <th scope="col">email</th>
+                    <th scope="col">roomId</th>
+                    <th scope="col">checkInDate</th>
+                    <th scope="col">checkOutDate</th>
+                    <th scope="col">totalNight</th>
+                </tr>
+            </thead>
+            <tbody>
+                {props.results.map((bookingData, index) => {
+                    return (
+                        <tr key={index} >
+                            <th key={"id" + index}>{bookingData.id}</th>
```

Comment:
As long as the `tr` element has a `key`, I don't think its children need keys, though it doesn't hurt to have them here :)